curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-13
 
 # --description--
 
-Target unordered list elements within `nav` elements, and use _Flexbox_ to evenly space the children.
+Use the `>` selector to target unordered list elements within `nav` elements, and use _Flexbox_ to evenly space the children.
 
```

Comment:
[Proofreading] Adding the definite article "the" would help campers gain a better understanding of the elements that need the selector. 
```suggestion
Use the `>` selector to target the unordered list elements within `nav` elements, and use _Flexbox_ to evenly space the children.

```